packages/wagmi/package.json

```diff
@@ -31,8 +30,7 @@
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "tsup": "^6.7.0",
-    "viem": "1.19.11",
-    "wagmi": "1.4.7"
+    "wagmi": "2.5.6"
```

Comment:
Shouldn't wagmi also be moved to regular dependencies?

Reply:
Oh I actually did not notice they were in both, it should only be in regular no ?